src/histograms/charts/AbstractChart.ts

```diff
@@ -562,13 +574,19 @@ export abstract class AbstractChart extends AbstractHistogram {
     const hoveredBuckets = data.filter(b => +b.key <= xDomainValue && +b.key > xDomainValue - dataInterval);
     const ys = [];
     let x;
+    let xEndDate;
+    let xStartDate;
     hoveredBuckets.forEach(hb => {
       if (HistogramUtils.isValueValid(hb)) {
         let color;
         if (!!hb.chartId && !!this.histogramParams.colorGenerator) {
           color = this.histogramParams.colorGenerator.getColor(hb.chartId);
         }
+
         x = HistogramUtils.toString(hb.key, this.histogramParams, dataInterval);
+        const calculatedTimeStamp = Math.floor(dataInterval)+(+hb.key);
```

Comment:
Math.floor is not to be applied here (we did it last time to simulate a time test) but should not be generalised